packages/validation-manager/src/ValidationManager.ts

```diff
@@ -327,4 +333,147 @@ export class ValidationManager implements IValidationManager {
   async getOperationHash (userOp: OperationBase): Promise<string> {
     return await this.entryPoint.getUserOpHash(packUserOp(userOp as UserOperation))
   }
+
+  // todo fix rest of the code to work with the new tracer result instead of adjusting it here
+  convertTracerResult (tracerResult: any, userOp: UserOperation): BundlerTracerResult {
+    const SENDER_CREATOR = '0xefc2c1444ebcc4db75e7613d20c6a62ff67a167c'.toLowerCase()
```

Comment:
`SENDER_CREATOR` really should be configurable, there may be sub-versions of EntryPoint and rebuilding the bundler for that will become a pain.